Client/mods/deathmatch/logic/CClientBuilding.cpp

```diff
@@ -97,13 +107,47 @@ void CClientBuilding::Create()
     ConvertZXYEulersToQuaternion(m_vRot, vRot4D);
 
     m_pBuilding = g_pGame->GetPools()->GetBuildingsPool().AddBuilding(this, m_usModelId, &m_vPos, &vRot4D, m_interior);
+
+    if (m_pHighBuilding)
+    {
+        dynamic_cast<CBuilding*>(m_pHighBuilding->GetGameEntity())->SetLod(m_pBuilding);
```

Comment:
If you want to use dynamic_cast please, check results before using them.

Reply:
...or avoid case